lib/beaker/host/unix/pkg.rb

```diff
@@ -27,8 +27,12 @@ def check_for_package(name, opts = {})
       when /sles-10/
         result = execute("zypper se -i --match-exact #{name}", opts) { |result| result }
         result.stdout =~ /No packages found/ ? (return false) : (return result.exit_code == 0)
-      when /sles-/
-        result = execute("zypper se -i --match-exact #{name}", opts) { |result| result }
+    when /sles-/
+        if !self[:sles_rpmkeys_nightly_pl_imported]
+          execute('rpmkeys --import http://nightlies.puppetlabs.com/07BB6C57', opts)
+          self[:sles_rpmkeys_nightly_pl_imported] = true
```

Comment:
Is this `sles_rpmkeys_nightly_pl_imported` read anywhere else? It might be nice to have some code comments to explain this key's significance and when a user needs to check its value.